### PR TITLE
Add coalesced events to eventRate example

### PR DESCRIPTION
**Pull request by NavidZ**
Add coalesced events to the drawing if the
user agent supports pointerevents and
getCoalescedEvents API is supported.

### CODE REVIEW

File: eventRate.js [hunk 1]
```diff
@@ -50,6 +56,7 @@ function startDraw()
 {
   startTime = performance.now();
   eventCount = 0;
+  coalescedEventCount = 0;
```
Comment (RByers) [inline eventRate.js]: Doesn't look like this is used for anything, remove?

File: eventRate.js [hunk 2]
```diff
@@ -63,56 +70,92 @@ function endDraw()
   startTime = undefined;
 }
 
-function addPoint(x, y)
+function addCoalescedPoint(x, y)
 {
   eventCount++;
```
Comment (RByers) [inline eventRate.js]: this line is in the wrong place now - eventCount is used for the event rate which I think we want to represent the real rate (not the coalesced rate).

File: eventRate.js [hunk 3]
```diff
@@ -63,56 +70,92 @@ function endDraw()
...
-  canvas.addEventListener('mousedown', function(e) {
-    if (e.button == 0) {
-      mouseDown = true;
-      startDraw();
-      addPoint(e.pageX, e.pageY);
+  if (window.PointerEvent) {
+    var primaryButtonDown = false;
+
+    canvas.addEventListener('pointerdown', function(e) {
+      if (e.button == 0) {
```
Comment (RByers) [inline eventRate.js]: nit: use e.isPrimary?

File: eventRate.js (hunk 1 above)
Comment (NavidZ) [inline eventRate.js]: Yeah. I wanted to show the coalesced event rate as well but I forgot. Now looking at your other comment it seems only the rate of the real events matter. So I just removed it.

File: eventRate.js (hunk 3 above)
Comment (NavidZ) [inline eventRate.js]: done

File: eventRate.js (hunk 2 above)
Comment (NavidZ) [inline eventRate.js]: Sure. I restore the original calculation for the frequency.

**Comment by RByers** [comment]
Thank you!

**Pull request merged**